src/domain/vesolje.py

```diff
@@ -7,8 +7,14 @@
 
 @dataclass
 class Vesolje:
+    st_vesoljcev: int = 8
+    razmik_vesoljcev_x: int = 0.1
+    razmik_vesoljcev_y: int = 0.1
+    premik_vesoljca_navzdol: float = 0.005
+    velikost_vesoljca_x: float = 0.08
+    velikost_vesoljca_y: float = 0.08
```

Comment:
Kršitev arhitekture programa... To vse spada v vesoljca LOL.

![download](https://user-images.githubusercontent.com/8678755/226456233-2c5e6df9-4be2-4b2b-9166-55b8489bb088.jpeg)